ja/browser/browser/newtab/asrouter.ftl

```diff
@@ -152,9 +166,23 @@ cfr-doorhanger-socialtracking-ok-button = トラッキング防止機能を確
   .accesskey = P
 cfr-doorhanger-socialtracking-close-button = 閉じる
   .accesskey = C
+cfr-doorhanger-socialtracking-dont-show-again = このようなメッセージは次から表示しない
+  .accesskey = D
 cfr-doorhanger-socialtracking-heading = { -brand-short-name } はソーシャルネットワークからの追跡を停止させました
 cfr-doorhanger-socialtracking-description = プライバシーは重要です。{ -brand-short-name } は一般的なソーシャルメディアトラッカーをブロックしました。オンラインでのあなたの行動を収集できるデータ量を制限しています。
 cfr-doorhanger-fingerprinters-heading = { -brand-short-name } はこのページのフィンガープリント採取をブロックしました
 cfr-doorhanger-fingerprinters-description = プライバシーは重要です。{ -brand-short-name } はあなたを追跡するために端末から一意に識別可能な情報を収集する、フィンガープリント採取をブロックしました。
 cfr-doorhanger-cryptominers-heading = { -brand-short-name } はこのページの暗号通貨マイニングをブロックしました
 cfr-doorhanger-cryptominers-description = プライバシーは重要です。{ -brand-short-name } は暗号通貨の採掘のためにあなたのシステムの計算リソースを利用する、暗号通貨マイニングをブロックしました。
+
+## Enhanced Tracking Protection Milestones
+
+# Variables:
+#   $blockedCount (Number) - The total count of blocked trackers. This number will always be greater than 1.
+#   $date (String) - The date we began recording the count of blocked trackers
+cfr-doorhanger-milestone-heading =
+  { $blockedCount ->
+   *[other] { -brand-short-name } は { $date } から <b>{ $blockedCount } 個</b>のトラッカーをブロックしました！
```

Comment:
{ -brand-short-name } blocked over <b>{ $blockedCount }</b> trackers since { $date }!
→  { $date } 以降、{ -brand-short-name } は <b>{ $blockedCount } 個</b>以上のトラッカーをブロックしました！